packages/middleware/build.js

```diff
@@ -4,14 +4,17 @@ const execa = require('execa');
 const { join } = require('path');
 
 async function main() {
+  const srcDir = join(__dirname, 'src');
   const outDir = join(__dirname, 'dist');
 
   // Start fresh
   await fs.remove(outDir);
 
-  await execa('tsc', [], {
-    stdio: 'inherit',
-  });
```

Comment:
Do we export any types from this package?

That might be a reason to keep `tsc` but I'm guessing its safe to remove

Reply:
Nope, the previous tsc command was actually changed to by me when the websandbox was a separate package. Didn't have enough context into how this was built at the time.